rules-base/api/src/main/java/org/jboss/windup/rules/files/FileDiscoveredListenerUtil.java

```diff
@@ -0,0 +1,27 @@
+package org.jboss.windup.rules.files;
+
+/**
+ * @author <a href="mailto:jesse.sightler@gmail.com">Jesse Sightler</a>
+ */
+public class FileDiscoveredListenerUtil
+{
+    public static boolean shouldSkip(Iterable<FileDiscoveredListener> listeners, FileDiscoveredEvent event)
+    {
+        boolean skipFile = false;
+        for (FileDiscoveredListener listener : listeners)
+        {
+            FileDiscoveredResult result = listener.fileDiscovered(event);
```

Comment:
well, speaking about ".class" file, why not to have a rule that runs just after <mapping>s are done over all of the .class files?